pkg/manager/manager.go

```diff
@@ -488,7 +508,7 @@ func (m *jobManager) estimateCompletion(requestedAt time.Time) time.Duration {
 	return lastEstimate.Truncate(time.Second)
 }
 
-func contains(arr []string, s string) bool {
+func Contains(arr []string, s string) bool {
```

Comment:
This method feels like it belongs in a more common/shared place since it's used in multiple packages, but really has nothing to do, specifically, with the `manager`

Reply:
Moved the function to `utils.go`